lib/ZwaveClient.js

```diff
@@ -899,13 +922,14 @@ function initNode (zwaveNode) {
   node.productType = zwaveNode.productType
   node.manufacturer = deviceConfig.manufacturer
   node.firmwareVersion = zwaveNode.firmwareVersion
-  node.zwaveVersion = zwaveNode.version
+  node.protocolVersion = zwaveNode.protocolVersion
   node.zwavePlusVersion = zwaveNode.zwavePlusVersion
-  node.nodeType = zwaveNode.nodeType
-  node.roleType = zwaveNode.roleType
+  node.nodeType = zwaveNode.zwavePlusNodeType
+  node.roleType = zwaveNode.zwavePlusRoleType
   node.endpointsCount = zwaveNode.getEndpointCount()
   node.isSecure = zwaveNode.isSecure
-  node.isBeaming = zwaveNode.isBeaming
+  node.supportsSecurity = zwaveNode.supportsSecurity
+  node.supportsBeaming = zwaveNode.supportsBeaming
   node.isControllerNode = zwaveNode.isControllerNode()
   node.isListening = zwaveNode.isListening
   node.isFrequentListening = zwaveNode.isFrequentListening
```

Comment:
This is not just a boolean now, in case you're expecting that somewhere

Reply:
Nope, I'm using `isListening` in zwave graph